org_status/org_status.py

```diff
@@ -92,22 +101,39 @@ def get_argument_parser():
 
 
 def encode_repo_list(repo_data, encoder_name, styled):
-    encoders = get_all_supported_encoders()
+    encoders = get_all_supported_formatters()
     encoded_repo_list = None
 
     for encoder in encoders:
         if encoder.NAME == encoder_name:
             try:
                 encoded_repo_list = encoder().convert_repo_list_to_format(
                     repo_data)
+                return encoded_repo_list
             except NotImplementedError:
                 print(styled(
                         f'{encoder_name} does not support exporting results',
                         'red'))
-        else:
-            print(styled(f'unknown export format {encoder_name}', 'red'))
 
-    return encoded_repo_list
+    print(styled(f'unknown export format {encoder_name}', 'red'))
+
+
+def decode_repo_list(file_name, decoder_name, styled):
+    decoders = get_all_supported_formatters()
```

Comment:
In `encode_repo_list` this line is repeated again. I think it'd be better to just pass the formatters in as arguments. I'm sure there are other approaches to this too.

Reply:
I disagree with this. Passing it in as an argument would make the methods' signatures unnecessarily more complex. In addition, get_all_support_formatters() returns a static tuple, so there is a negligible difference in performance if we were to pass it in as an argument.